pyproject.toml

```diff
@@ -64,6 +64,9 @@ twine = "^4.0.1"
 opentelemetry = ["opentelemetry-api", "opentelemetry-sdk"]
 grpc = ["grpcio"]
 
+[tool.poetry.group.dev.dependencies]
+wheel = "^0.42.0"
```

Comment:
Can this be moved to the `tool.poetry.dev-dependencies` section above? I know that newer poetry has it like this, but for older ones (which some envs are forced to use I think), we already have a list above.